haproxy/test/controls/service_running.rb

```diff
@@ -0,0 +1,13 @@
+control 'ensure backend web process is running and owned by correct owner' do
+  describe processes(Regexp.new("httpd")) do
+    its('list.length') { should >= 1 }
+    its('users') { should include 'hab' }
```

Comment:
We might only need the `ensure backend web service is listening` control to give us evidence that the webapp is working. As long as it works well enough serve the testing of haproxy, we shouldn't care what state webapp is in.
So I suggest removing the `its("users")` assertion here; and possibly also this entire control.